aws/signer/v4/functional_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/aws/endpoints"
-	"github.com/aws/aws-sdk-go-v2/aws/signer/v4"
+	v4 "github.com/aws/aws-sdk-go-v2/aws/signer/v4"
```

Comment:
I've noticed this happening recently, is this a Go formater change? Is it noticing that we have a package that is called v4, but also might confusingly match import path versioning?

Reply:
:( `goimports` is doing this. I'm not positive why, but this is an artifact of that utility.

Reply:
I can revert this file though since there are no other changes here.